code_samples/datetime_classes_sample.js

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+// This sample code demonstrates datetime classes usage.
+
+const { Client, LocalDate, OffsetDateTime } = require('..');
+
+class CustomNumber {
```

Comment:
the name CustomNumber is not in line with the fields we have, right? Maybe we can rename it

Reply:
Same comment above, we don't use this. If you intend to just show the usage, without using it, I believe we need to document this with a comment. Also, in this case, I don't think we need the factory definition or config registration.

But I would like to see a simple usage, put it to some data structure (like set etc.)